src/ProjectService.Mappers/ResponsesMappers/Interfaces/IRoleExpandedResponseMapper.cs

```diff
@@ -0,0 +1,13 @@
+﻿using LT.DigitalOffice.ProjectService.Models.Db;
+using LT.DigitalOffice.ProjectService.Models.Dto.Responses;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace LT.DigitalOffice.ProjectService.Mappers.ResponsesMappers.Interfaces
+{
+    public interface IRoleExpandedResponseMapper
+    {
+        RoleExpandedResponse Map(DbRole dbRole);
+    }
```

Comment:
Why not IMapper<DbRole, RoleExpandedResponse>?

Reply:
Forgot about it, now added

Reply:
Explained in a discussion with @djino97 what was wrong